include/nuttx/net/net.h

```diff
@@ -59,6 +59,32 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
+/* Definitions of 8-bit socket flags */
```

Comment:
Move after line 110, so all socket flag in one location

Reply:
Done